src/addScore.js

```diff
@@ -0,0 +1,37 @@
+import {
+  form, user, score, apiURL,
+} from './variable.js';
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  async function addData() {
+    const add = await fetch(apiURL, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        user,
+        score,
+      }),
+    });
+    return add.json();
+  }
+  addData().then((cheeseData) => {
+    if (cheeseData) {
+      const successAlert = `<div class="alert alert-success" role="alert">
+            Score Added successfully! Please click on Refresh 
+            </div>`;
+      document
+        .querySelector('.score-box')
```

Comment:
| 🔴 Posting Data Issue |
|-|
| ![image](https://github.com/Mahdi-Niazi/leaderboard-project/assets/93039359/3b8ca22f-5612-471a-8f58-5425c91745ba) |

- [ ] Great work on getting the data from the API 👍 but I guess your post request is not being done properly 😨 Kindly fix this error because I cannot get any new data after posting the data 🤔 Kindly fix this issue to make your app work as expected and follow the project requirements 😊